File: Arrays&Hashing/ValidAnagram/Solution1.cs
public class Solution {
    public bool IsAnagram(string s, string t)
    {
        if (s.Length != t.Length)
            return false;

        var alphabet = new int[26];

        for (int i = 0; i < s.Length; i++)
        {
            alphabet[s[i] - 'a']++;
            alphabet[t[i] - 'a']--;
        }

        return alphabet.All(x => x == 0);
    }
}

File: Arrays&Hashing/ValidAnagram/Solution1.js
class Solution {
    /**
     * @param {string} s
     * @param {string} t
     * @return {boolean}
     */
    isAnagram(s, t) {
        if (s.length !== t.length) return false;
        var alphabet = new Array(26).fill(0);
        
        for (let index = 0; index < s.length; index++) {
            alphabet[s.charCodeAt(index) - 97]++;
            alphabet[t.charCodeAt(index) - 97]--;
        }

        return alphabet.every(x => x === 0);
    }
}
